.github/workflows/release-prepare-hotfix.yml

```diff
@@ -21,21 +32,57 @@ jobs:
         with:
           app_id: ${{ secrets.GH_BOT_APP_ID }}
           private_key: ${{ secrets.GH_BOT_PRIVATE_KEY }}
-      - name: Checkout Repo
+
+      - uses: actions/checkout@v4
+        with:
+          ref: support/post-ghost-release-hotfix-workflow-TEST  # CHANGE THIS, TESTING ONLY
+          sparse-checkout: tools/actions/composites
+
+      - name: Generate ref/tag version to use during main checkout
+        uses: ./tools/actions/composites/generate-tag
+        id: format-app-tag
+        with:
+          tag_version: ${{ github.event.inputs.tag_version }}
+          application: ${{ github.event.inputs.application }}
+
+      - name: Checkout Repo (develop branch)
         uses: actions/checkout@v4
         with:
-          ref: ${{ inputs.ref }}
+          ref: develop
```

Comment:
pinning the ref to develop is for testing purpose, am i wrong ?

Reply:
In fact we shouldn't be doing 2 separate steps for checkout and then fetch (below). We should just be checking out ${{ inputs.ref }} in the first place. Will update.